### ISSUE TITLE
Delay while taking sream

### ISSUE THREAD

**Issue by AndyKD** [yixia/VitamioBundle#163]
First Thanks to vitamio team.

While i,m playing live rtsp its taking delay of 4 to 5 seconds, please help me out to reduce the delay, please provide me some code or example to implement this..

**Comment by ghost** [comment]
@wangzhengyi, did it resolve your issue, setting the buffer size? In my case I am using Vitamio media player and have set the buffer size of 1000, which reduced the latency 2-3 seconds but after 5-10 mins, It starts increasing (10 sec). May be bcoz buffer is getting full (Not Sure). How can we flush/ clear its buffer any idea. Any help would be greatly appreciated.
Thanks.

**Comment by AndyKD** [comment]
@wangzhengyi, you are right. We are getting this same issue here also. I had use setUseCache(false), but delay is their after 10-15 mintues and its getting increasing over time.

**Comment by sreyasvpariyath** [comment]
I was having the same issue and I could manage it by calling seekTo to the current time. Whenever I am facing delay in stream I would call seekTo.
I don't know whether its the solution or not but I can now stream the rtsp almost real time. 
mVideoView.seekTo(System.currentTimeMillis());
